### ISSUE TITLE
[BUG] Kirby Menu: Does not close on wheel scroll + menu not usable using trigger 'hover' + keyboard accessibility not supported

### ISSUE THREAD

**Issue by PeterLarsenIT** [kirbydesign/designsystem#3257]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
1. The menu does not close when there is a wheel scroll event. (Please notice that this is the 'wheel' event end not the 'scroll' event)

2. When the menu is configured with [triggers]="'hover'", then the menu will open when the menu button is hovered. However moving the mouse to the menu will cause the menu to close - probably because the button is not hovered anymore.

3. It is not possible to navigate the menu using the keyboard. I also think opening the menu using the keyboard should trigger other menus to be closed - currently all opened menus are kept open, however I am not sure how that is expected to work.

It would be nice with an event when the menu is opened to be able to e.g. fetch data to decide what should be available in the menu.

## Describe how to reproduce the bug
See bug description.

## Which Kirby version was used?
8.11.1

## What was the expected behavior?
See bug description.

## Please complete the following information:
- OS: MacOS
- Browser: Microsoft Edge for Business
- Browser version: 117.0.2045.35
## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
